### PR TITLE
163176948 identity contract

### PR DESCRIPTION
**Pull request by sirpy**

### CODE REVIEW

File: src/server/server.config.js [hunk 1]
```diff
@@ -45,27 +47,24 @@ const conf = convict({
...
-    default: 'ropsten',
+    format: ["kovan", "mainnet", "rinkbey", "ropsten","truffle","ganache","fuse"],
+    default: 'kovan',
     env: "NETWORK"
   }
 });
 
 // Load environment dependent configuration
 const env = conf.get('env');
+console.log(networks)
```
Comment (dminones) [inline src/server/server.config.js]: This should be removed I think

File: src/server/verification/verificationAPI.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// @flow
+import passport from "passport"
+import { get } from 'lodash'
+import {type UserRecord, StorageAPI, VerificationAPI } from '../../imports/types'
+import AdminWallet from '../blockchain/AdminWallet'
+
+function wrapAsync(fn) {
```
Comment (dminones) [inline src/server/verification/verificationAPI.js]: Shouldn't this be imported from 'import { wrapAsync } from 'server-middlewares'?

Comment (sirpy) [reply src/server/verification/verificationAPI.js]: Yes

File: src/server/server.config.js (hunk 1 above)
Comment (sirpy) [reply src/server/server.config.js]: Yes.

**Comment by sirpy** [comment]
Please make the changes you suggested and merge

**Pull request merged**